docs/quickstart/google-cloud-platform.md

```diff
@@ -0,0 +1,116 @@
+# Google Cloud Platform
+
+In this quickstart, you will learn how to deploy the Coder control plane instance, and deploy your first template. 
+
+## Requirements
+
+This quickstart assumes you have `roles/compute.instanceAdmin.v1` access to your Google Cloud Platform project.  
+
+## Setting Up your VM  
+
+If this is the first time you’re creating a VM on this project, you will need to enable the `Compute Engine API`. On the Compute Engine API page, click `enable` and wait for the service to finish connecting.
+
+<img src="../images/quickstart/google-cloud-platform/gcp0.png">
+
+This will pull up the `Create an Instance` page - name the instance something relevant to this project, following your naming convention of choice. In addition, select a region and zone that is close to your physical location. For this instance, we will use the base suggested image.
+
+<img src="../images/quickstart/google-cloud-platform/gcp1.png">
+
+Under `Identity and API Access`, click `Allow full access to all Cloud APIs`. Scroll down to `Firewall` and click `Allow HTTPS traffic` and `Allow HTTP traffic`.
+
+<img src="../images/quickstart/google-cloud-platform/gcp2.png">
+
+Scroll down to the bottom and click `Create` to create and deploy the VM.
+
+Congrats you’ve created your VM instance!
+
+## SSH-ing into the VM
+
+On the Compute Engine Dashboard, click on the VM for this project. Under `Details`, click `SSH` and select `Open in browser window`.
```

Comment:
Nit: the rest of the tutorial doesn't depend on how the user enters a terminal. I think we can leave the instruction more open-ended. I, for one, always prefer native terminals over the web terminal.

Reply:
t- however for this, im getting the quickest way to get started, from 0. if someone wants to losely follow the tutorial and use their own native terminal go crazy ? but this tutorial is for the 0 :)

Reply:
kk